app/models/edo_oracle/financial_aid/queries.rb

```diff
@@ -576,6 +576,14 @@ def self.get_awards_by_term_by_type(person_id, aid_year, award_type)
         SQL
       end
 
+      def self.get_financial_resources_links()
+        result = safe_query <<-SQL
+        SELECT UC.URL_ID          AS URL_ID
+          FROM SYSADM.PS_UCC_LINKAPIURL UC
+        WHERE UC.URL_ID LIKE 'UC_FA_FINRES%'
```

Comment:
Indent this to make a clear difference between the SQL and HEREDOC markers.

Reply:
I dunno?  I'm just following the same pattern used on all the other defs in the file?